elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreHelper.java

```diff
@@ -15,43 +21,75 @@
 import java.util.LinkedHashMap;
 import java.util.LinkedHashSet;
 import java.util.Map;
-import java.util.Optional;
 import java.util.Set;
 
+/**
+ * Helper for Aggregation Data Store which does the work associated with extracting {@link Query}.
+ */
 public class AggregationDataStoreHelper {
 
+    private static final int AGGREGATION_METHOD_INDEX = 0;
```

Comment:
Add a `TODO` comment to mark we are gonna to support selecting aggregation method in the future.

Reply:
Done